Make tabbed layout behave like a vertical container

This makes up/down cycle through tabs instead of left/right

--- sway-1.7.orig/sway/commands/focus.c
+++ sway-1.7/sway/commands/focus.c
@@ -24,10 +24,10 @@
 	if (strcasecmp(name, "prev") == 0) {
 		switch (parent_layout) {
 		case L_HORIZ:
-		case L_TABBED:
 			*out = WLR_DIRECTION_LEFT;
 			break;
 		case L_VERT:
+		case L_TABBED:
 		case L_STACKED:
 			*out = WLR_DIRECTION_UP;
 			break;
@@ -39,10 +39,10 @@
 	} else if (strcasecmp(name, "next") == 0) {
 		switch (parent_layout) {
 		case L_HORIZ:
-		case L_TABBED:
 			*out = WLR_DIRECTION_RIGHT;
 			break;
 		case L_VERT:
+		case L_TABBED:
 		case L_STACKED:
 			*out = WLR_DIRECTION_DOWN;
 			break;
@@ -163,12 +163,12 @@
 		list_t *siblings = container_get_siblings(current);
 
 		if (dir == WLR_DIRECTION_LEFT || dir == WLR_DIRECTION_RIGHT) {
-			if (parent_layout == L_HORIZ || parent_layout == L_TABBED) {
+			if (parent_layout == L_HORIZ) {
 				can_move = true;
 				desired = idx + (dir == WLR_DIRECTION_LEFT ? -1 : 1);
 			}
 		} else {
-			if (parent_layout == L_VERT || parent_layout == L_STACKED) {
+			if (parent_layout == L_VERT || parent_layout == L_STACKED || parent_layout == L_TABBED) {
 				can_move = true;
 				desired = idx + (dir == WLR_DIRECTION_UP ? -1 : 1);
 			}
--- sway-1.7.orig/sway/commands/move.c
+++ sway-1.7/sway/commands/move.c
@@ -79,9 +79,9 @@
 static bool is_parallel(enum sway_container_layout layout,
 		enum wlr_direction dir) {
 	switch (layout) {
-	case L_TABBED:
 	case L_HORIZ:
 		return dir == WLR_DIRECTION_LEFT || dir == WLR_DIRECTION_RIGHT;
+	case L_TABBED:
 	case L_STACKED:
 	case L_VERT:
 		return dir == WLR_DIRECTION_UP || dir == WLR_DIRECTION_DOWN;
